src/index.js

```diff
@@ -48,7 +48,8 @@ export default function regionHelper(name, options) {
             regionClass: regionClass,
             parentEl: function () {
                 return view.$el;
-            }
+            },
+            hash: hash
```

Comment:
Why not use [short notation](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015)?